typedload/datadumper.py

```diff
@@ -57,20 +57,29 @@ def __init__(self):
 
         self.hidedefault = True
 
+        self.handlers = [
+            (lambda value: type(value) in self.basictypes, lambda l, value: value),
+            (lambda value: isinstance(value, tuple) and hasattr(value, '_fields') and hasattr(value, '_asdict'), _namedtupledump),
+            (lambda value: isinstance(value, (list, tuple, set)), lambda l, value: [l.dump(i) for i in value]),
+            (lambda value: isinstance(value, Enum), lambda l, value: l.dump(value.value)),
+            (lambda value: isinstance(value, Dict), lambda l, value: {l.dump(k): l.dump(v) for k, v in value.items()}),
+        ]
```

Comment:
You could add a docstring about the cond stuff here as well.

Reply:
Done